apps/els_lsp/test/els_code_action_SUITE.erl

```diff
@@ -14,6 +14,7 @@
         , export_unused_function/1
         , suggest_variable/1
         , fix_module_name/1
+        , action_remove_macro/1
```

Comment:
Nit: since all the test cases in this suite relate to code actions, we can skip the `action_` prefix.

Reply:
address in c3849b7